_content/monographs/phra-malai-legend-thai-buddhist-literature_brereton-bonnie-pacala.md

```diff
@@ -0,0 +1,24 @@
+---
+title: "The Phra Malai legend in Thai Buddhist literature: A study of three texts"
+authors:
+  - "Bonnie Pacala Brereton"
+external_url: "https://deepblue.lib.umich.edu/handle/2027.42/128920"
+drive_links:
+  - "https://drive.google.com/file/d/16zOKTreDWyNRGcKqeAXFTDI-LxNwdtYy/view?usp=sharing"
+course: theravada-roots
+tags:
+  - thai-roots
```

Comment:
Let's add `thai-ideology` as well.

And yeah, go ahead and add the `myth` tag too (which is for Buddhist myths. The `mythology` tag is for cross cultural mythology studies. The beauty of these shadow tags is they are hidden on the site, so they can be a bit messy 😊  Just add them in and we can clean up those tags later when it's time to add them ☺️)